tests/Platforms/DB2PlatformTest.php

```diff
@@ -128,7 +128,8 @@ public function getAlterTableColumnCommentsSQL(): array
     {
         return [
             'ALTER TABLE mytable ' .
-            'ADD COLUMN quota INTEGER NOT NULL WITH DEFAULT',
+            'ADD COLUMN quota INTEGER NOT NULL WITH DEFAULT ' .
```

Comment:
See https://github.com/doctrine/dbal/blob/3.7.x/tests/Platforms/AbstractPlatformTestCase.php#L432-L440
To understand why those tests where incorrect